.storybook/webpack.config.js

```diff
@@ -16,6 +16,17 @@ module.exports = ({config, mode}) => {
     rule => !/js\|mjs\|jsx\|ts\|tsx/.test(rule.test.toString())
   );
 
+  // Override CRA postcss presets
+  config.module.rules.forEach(rule => {
+    if (rule.test.toString().includes('scss|sass')) {
+      rule.use[2].options.plugins = () => [
+        require('postcss-discard-duplicates'),
+        require('autoprefixer'),
+        require('postcss-inline-svg'),
+      ];
+    }
+  });
+
```

Comment:
@anicholls so this is quite the hack but what's going on is when we use `react-app` as a preset we are unable to override their default config.

see: https://github.com/facebook/create-react-app/issues/5749

Reply:
Good find and fix! I think this is fine for now 👍

Reply:
Though I wonder if there's any way to reference the `.postcss.json` file here rather than just copying the plugins from the `development` key...

Reply:
Yup, I'll look into it 👍

Reply:
@anicholls Updated to just pass in config path.